src/rust/engine/process_execution/src/lib.rs

```diff
@@ -130,6 +130,11 @@ pub struct ExecuteProcessRequest {
   #[derivative(PartialEq = "ignore", Hash = "ignore")]
   pub description: String,
 
+  // This will be materialized for local ExecuteProcessRequest only.
+  // Eventually we want to remove this.
+  // Context: https://github.com/pantsbuild/pants/pull/8282
+  // Think twice before using it.
+  pub local_only_scratch_files: hashing::Digest,
```

Comment:
This is a nit, but the way it's worded, it makes me think that the issue link will point to an issue tracking the removal of the scratch file, not the description motivating these changes.

If you could change the message to point to another issue saying when to remove it as well as what's already there, it would be awesome.

Reply:
+1

Also, please rename this to reflect whatever name we settle on for the python side.

Reply:
with https://github.com/pantsbuild/pants/issues/8314